_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+---
+layout:    post
+title:     "10 praw użytecznego designu"
+published: true
+date:      2019-07-09 23:58:00 +0100
+author:    pswierzko
+tags:
+    - ux
+    - ui
+    - design
+---
+
+W latach 80. Dieter Rams zagubiony w chaosie form i kolorów postanowił coś z tym zrobić. Doszedł do wniosku, że dobry design nie może trzymać się sztywnych ram, ale opierając się na pewnych regułach, można stworzyć dobry produkt. Idąc tym tokiem rozumowania opracował 10 reguł dobrego designu, które w branży zdążyły urosnąć do rangi dziesięciu przykazań, a on sam uchodzi dziś za jednego z najlepszych i najbardziej wpływowych projektantów XX wieku.
+
+Choć świat UX designu nie doczekał się swojego Rams'a nie oznacza to, że projektanci interfejsów i doświadczeń użytkowników zostali pozostawieni na pastwę losu. Jak się sami za chwilę przekonacie, niezliczone ilości badań na przestrzeni ostatnich kilkudziesięciu lat zaowocowały całkiem pokaźną listą maksym i reguł.
+
+## 1. Efekt „estetycznej funkcjonalności”
+
+W latach 90 zeszłego stulecia Masaaki Kurosu i Kaori Kashimura postanowili zbadać wpływ atrakcyjności estetycznej na sposób, w jaki postrzega się funkcjonalność. W tym celu poprosili 252 uczestników badania o to, aby ocenili 26 różnych bankomatów, który z nich jest najłatwiejszy w obsłudze, a który najładniejszy. W ten sposób Masaki i Kaori zaobserwowali, że istnieje pewna korelacja - im coś jest piękniejsze tym jest bardziej użyteczne. Dodatkowo piękny wygląd sprawia, że jesteśmy skłonni wybaczyć drobne błędy użyteczności, czar jednak pryska jak bańka mydlana, gdy natrafimy na dużą wadę jak np. brak możliwości złożenia zamówienia.
+
+Spróbujecie przypomnieć sobie sytuację, kiedy musieliście podjąć decyzję i wybrać produkt spośród kilku dostępnych? Jeśli wybraliście coś tylko dlatego, że miało ładną etykietę, to właśnie ulegliście efektowi estetycznej funkcjonalności:)
+
+<div class="img-with-legend">
+<img alt="Strona główna firmy Apple" src="/assets/img/posts/2019-07-09-design/1.jpg" />
+<span class="img-legend">Rys. 1. Strona główna firmy Apple</span>
+</div>
+
+## 2. Prawo Jakoba
+
+Jakob Nielsen, jeden z czołowych specjalistów w dziedzinie użyteczności, a także założyciel Nielsen Norman Group zauważył, że użytkownicy spędzają większość czasu na innych stronach. Oznacza to, że użytkownicy wolą, aby Twoja strona działała tak samo, jak wszystkie inne witryny, które już znają. Dobrym przykładem wykorzystania tej zasady są sklepy sprzedające odzież i obuwie on-line.
```

Comment:
Tu bym coś zmienil, coś nie gra:
"użytkownicy spędzają większość czasu na innych stronach. Oznacza to, że użytkownicy wolą, aby Twoja strona działała tak samo, jak wszystkie inne witryny, które już znają" 

Jedno z drugiego wcale nie wynika przyczyn może być więcej:
 "innych stronach" - jest więcej (jeśli liczymy je razem)
jeśli mówimy że więcej czasu na jakiejś konkretnej pojedyńczej stronie to też przyczyn może być więcej:
lepsze ceny, rabaty, szersza oferta, a nawet mniej wygodna strona co zmusza do większej ilości czasu.
 
Jakieś jeszcze jedno zdanie doprecyzowania by się przydało.